test/features/users/sign_out_a_user_test.rb

```diff
@@ -0,0 +1,12 @@
+require 'test_helper'
+
+feature 'Signing out user' do
+  before do
+    sign_in
+  end
+
+  scenario 'returns a sucess message' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.